kytos-amlight-topo-init.sh

```diff
@@ -0,0 +1,172 @@
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:12:12 '{"mtu": 9000, "port_name": "Interface-12:12"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:12:4  '{"mtu": 9000, "port_name": "Interface-12:4"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:12:8  '{"mtu": 9000, "port_name": "Interface-12:8"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:13:17 '{"mtu": 9000, "port_name": "Interface-13:17"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:13:2  '{"mtu": 9000, "port_name": "Interface-13:2"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:13:3  '{"mtu": 9000, "port_name": "Interface-13:3"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:13:5  '{"mtu": 9000, "port_name": "Interface-13:5"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:14:7  '{"mtu": 9000, "port_name": "Interface-14:7"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:14:8  '{"mtu": 9000, "port_name": "Interface-14:8"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:15:4  '{"mtu": 9000, "port_name": "Interface-15:4"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:15:6  '{"mtu": 9000, "port_name": "Interface-15:6"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:15:7  '{"mtu": 9000, "port_name": "Interface-15:7"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:16:5  '{"mtu": 9000, "port_name": "Interface-16:5"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:16:6  '{"mtu": 9000, "port_name": "Interface-16:6"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:17:10 '{"mtu": 9000, "port_name": "Interface-17:10"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:17:9  '{"mtu": 9000, "port_name": "Interface-17:9"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:11 '{"mtu": 9000, "port_name": "Interface-18:11"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:13 '{"mtu": 9000, "port_name": "Interface-18:13"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:14 '{"mtu": 9000, "port_name": "Interface-18:14"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:16 '{"mtu": 9000, "port_name": "Interface-18:16"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:19:12 '{"mtu": 9000, "port_name": "Interface-19:12"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:19:13 '{"mtu": 9000, "port_name": "Interface-19:13"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:19:15 '{"mtu": 9000, "port_name": "Interface-19:15"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:20:16 '{"mtu": 9000, "port_name": "Interface-20:16"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:20:17 '{"mtu": 9000, "port_name": "Interface-20:17"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:21:14 '{"mtu": 9000, "port_name": "Interface-21:14"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:21:18 '{"mtu": 9000, "port_name": "Interface-21:18"}'
```

Comment:
I've pushed this commit https://github.com/amlight/kytos-end-to-end-tests/pull/132/commits/5fa222260bb9b7fbcebd750aa04262dfde61e2a3, based on the topology, it looks like it was meant to be 21:18 instead of 21:15

Reply:
yes, you are right @viniarck